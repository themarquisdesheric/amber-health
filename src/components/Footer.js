import React from 'react';

import instagram from '../img/social/instagram.svg';
import twitter from '../img/social/twitter.svg';

const Footer = () => (
  <footer className="footer pt-12">
    <small className="flex justify-center text-center py-4">
      <span>
        Â© {new Date().getFullYear()} Amber Robinson
      </span>
      <span className="social">
        <a title="instagram" href="https://instagram.com" className="px-3">
          <img
            src={instagram}
            alt="Instagram"
          />
        </a>
        <a title="twitter" href="https://twitter.com">
          <img
            src={twitter}
            alt="Twitter"
          />
        </a>
      </span>
    </small>
  </footer>
);

export default Footer;
