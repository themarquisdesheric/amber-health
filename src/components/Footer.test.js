import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from './Footer';

test('Footer', () => {
  const wrapper = shallow(<Footer />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
