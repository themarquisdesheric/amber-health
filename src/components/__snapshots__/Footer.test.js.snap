// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer
  className="footer relative pt-12"
>
  <SocialLinks />
  <small
    className="flex justify-center py-4"
  >
    <span>
      Â© 
      2019
       Amber Robinson
    </span>
  </small>
</footer>
`;
